src/node_api.cc

```diff
@@ -673,6 +673,38 @@ v8::Local<v8::Object> CreateAccessorCallbackData(napi_env env,
   return cbdata;
 }
 
+// Pointer used to identify items wrapped by N-API. Used by FindWrapper and
+// napi_wrap().
+char napi_wrap_name[] = "N-API Wrapper";
```

Comment:
I left out the `const` because converting from `const char*` to `void*` requires a pile of casting:
```C++
reinterpret_cast<void*>(const_cast<char*>(v8impl::napi_wrap_name))
```
But anyway, I'll add it.